model/src/pyrenew/latent/hospitaladmissions.py

```diff
@@ -14,7 +14,7 @@
 
 class HospAdmissionsSample(NamedTuple):
     """
-    A container for holding the output from HospAdmissionsSample.sample.
+    A container for holding the output from `lantent.HospAdmissions.sample()`.
```

Comment:
"latent" is spelled incorrectly.

"from [the] ..." works better.

Reply:
I'd omit the "the" and keep the function call notation. Perhaps:

```suggestion
    A container to hold the output of `latent.HospAdmissions.sample()`.
```

Reply:
Addressed in 439367c